<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_login</class>
 <widget class="QDialog" name="fm_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#ececec
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_user">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>User</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_user">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_senha">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>Senha</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_senha">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cbMostrarSenha">
        <property name="styleSheet">
         <string notr="true">QCheckBox{
color:#303030;
font-size:9pt;
}</string>
        </property>
        <property name="text">
         <string>Mostrar senha</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cbSalvarUser">
        <property name="styleSheet">
         <string notr="true">QCheckBox{
color:#303030;
font-size:9pt;
}</string>
        </property>
        <property name="text">
         <string>Salvar user</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="vs">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:enabled{background:#faa91d;}
QPushButton:disabled{background:#fabe50;}
QPushButton:pressed{background:#ffd500;}
QPushButton{
color:#ffffff;
font-size:9pt;
border:none;
border-radius:10px;
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_logar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:enabled{background:#7dbe50;}
QPushButton:disabled{background:#aaff7f;}
QPushButton:pressed{background:#aaff00;}
QPushButton{
color:#ffffff;
font-size:9pt;
border:none;
border-radius:10px;
}</string>
          </property>
          <property name="text">
           <string>Logar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_user</tabstop>
  <tabstop>line_senha</tabstop>
  <tabstop>btn_logar</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
